### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
 
 ![Atom](https://cloud.githubusercontent.com/assets/72919/2874231/3af1db48-d3dd-11e3-98dc-6066f8bc766f.png)
 
-Most of the Atom user and developer documentation is contained in the [Atom Docs](https://github.com/atom/docs) repository.
+Most of the Atom user and developer documentation is contained in the [Atom Flight Manual](https://github.com/atom/flight-manual.atom.io) repository.
 
 In this directory you can only find very specific build and API level documentation. Some of this may eventually move to the docs repository as well.
```
Comment (50Wliu) [inline docs/README.md]: > Some of this may eventually move to the **Flight Manual** as well.

**Comment by torn4dom4n** [comment]
@50Wliu I replaced it.

**Comment by 50Wliu** [comment]
Sorry...when I bolded Flight Manual, I did that so it would be easier for you to see what needed to be replaced, not to actually bold it in the readme 🙈.  My fault!

**Comment by torn4dom4n** [comment]
I think it is better now.

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**